projects/github-actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -255,20 +240,35 @@ async function getStatusChecks( payload, octokit ) {
 	const ownerLogin = owner.login;
 
 	const hasLongDescription = body?.length > 200;
-	const isLabeled = await hasStatusLabels( octokit, ownerLogin, repo, number );
 	const hasTesting = !! body?.includes( 'Testing instructions' );
 	const hasPrivacy = !! body?.includes( 'data or activity we track or use' );
 	const projectsWithoutChangelog = await getChangelogEntries( octokit, ownerLogin, repo, number );
 	const isFromContributor = head.repo.full_name === base.repo.full_name;
 
+	const prLabels = await getLabels( octokit, ownerLogin, repo, number );
+	const { hasStatusLabels, hasTypeLabels } = prLabels.reduce(
+		( acc, label ) => {
+			// We're only interested in status labels, but not the "Needs Reply" label since it can be added by the action.
+			if ( label.match( /^\[Status\].*(?<!Author Reply)$/ ) ) {
+				acc.hasStatusLabels = true;
+			}
+			if ( label.match( /^\[Type\]/ ) ) {
+				acc.hasTypeLabels = true;
+			}
+			return acc;
+		},
+		{ hasStatusLabels: false, hasTypeLabels: false }
+	);
+
 	return {
 		hasLongDescription,
-		isLabeled,
+		hasStatusLabels,
 		hasTesting,
 		hasPrivacy,
 		projectsWithoutChangelog,
 		hasChangelogEntries: projectsWithoutChangelog.length === 0,
 		isFromContributor,
+		hasTypeLabels,
```

Comment:
I'd probably move this up next to `hasStatusLabels`, but it doesn't really matter.

Reply:
Done in 0dc30d4211e7deb14c5830642dce5242a93f9951